html-forms/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+      HTML Forms
+    </title>
+  </head>
+  <body>
+    <h1>
+      Coding Form!
+    </h1>
+    <form action="index.html">
+      <div>
+        Email
+        <br />
+      </div>
+      <label for="email">
+      <input id="email" type="text" name="email" size="20" maxlength="40" />
+      </label>
+    </form>
+    <form action="index.html">
+      <div>
+        Why is coding cool?
+        <br />
+      </div>
+      <select name="choices">
+        <option id="sel" value="select" selected="selected">
+          <label for="sel">Select an option</label>
+        </option>
+        <option id="inter" value="interesting">
+          <label for="inter">Because it's interesting</label>
+        </option>
+        <option id="fun" value="fun">
+          <label for="fun">Because it's fun</label>
+        </option>
+        <option id="power" value="powerful">
+          <label for="power">Because it's powerful</label>
+        </option>
+      </select>
+    </form>
+    <form action="index.html">
+      <div>
+        Favorite Language (choose one)
+        <br />
+      </div>
+      <input id="HT" type="radio" name="language" value="HTML" checked="checked" />
+      <label for="HT">HTML</label>
+      <input id="CS" type="radio" name="language" value="CSS" />
+      <label for="CS">CSS</label>
+      <input id="JS" type="radio" name="language" value="JavaScript" />
+      <label for="JS">JavaScript</label>
+    </form>
+    <form action="index.html">
+      <div>
+        Would you like to sign up for more information?
+        <br />
+      </div>
+      <input id="sign" type="checkbox" name="choice" value="signmeup" />
+      <label for="sign">Sign me up!</label>
+    </form>
+    <form action="index.html">
+      <div>
+        Comments
+        <br />
+      </div>
+      <label for="comm">
```

Comment:
Put this label around the label text for this input, the "Comments" text on the line above 👍